tests/test_nih.py

```diff
@@ -15,12 +15,49 @@ def test_host(self):
     def test_title(self):
         self.assertEqual(self.harvester_class.title(), "Baked Tilapia With Tomatoes")
 
+    def test_description(self):
+        self.assertEqual(
+            "This dish is easy to prepare, low in calories, and economical.",
+            self.harvester_class.description(),
+        )
+
+    def test_recipe_source(self):
+        self.assertEqual(
+            "Delicious Heart Healthy Latino Recipes",
+            self.harvester_class.recipe_source(),
+        )
+
     def test_total_time(self):
         self.assertEqual(35, self.harvester_class.total_time())
 
+    def test_prep_time(self):
+        self.assertEqual(10, self.harvester_class.prep_time())
+
+    def test_cook_time(self):
+        self.assertEqual(25, self.harvester_class.cook_time())
+
+    def test_serving_size(self):
+        self.assertEqual("1 fillet", self.harvester_class.serving_size())
+
     def test_yields(self):
         self.assertEqual("4 servings", self.harvester_class.yields())
 
+    def test_nutrients(self):
+        self.assertCountEqual(
```

Comment:
```suggestion
        self.assertEqual(
```

(perhaps a little more fragile if the website content changes -- but this is generally consistent with the assertions we use for `test_nutrients` test cases, so I think it'd make sense to use here too)

Reply:
Change completed

Reply:
Thanks!